test/index.js

```diff
@@ -23,12 +23,18 @@ const mockHook = (delay = 0) => ({
     this.taps.map(t => setTimeout(() => t.cb(t.name), delay));
   }
 });
+const mockLogger = {
+  info: fake(),
+  error: fake()
+};
```

Comment:
These are the mock logging methods to verify, so they are `fake()`s to capture calls and arguments.